example/data/collections/s3_MOD09GQ_006.json

```diff
@@ -6,7 +6,8 @@
 	"provider_path": "cumulus-test-data/pdrs",
 	"duplicateHandling": "replace",
 	"granuleId": "^MOD09GQ\\.A[\\d]{7}\\.[\\S]{6}\\.006.[\\d]{13}$",
-	"granuleIdExtraction": "(MOD09GQ\\..*)(\\.hdf|\\.cmr|_ndvi\\.jpg)",
+  "granuleIdExtraction": "(MOD09GQ\\..*)(\\.hdf|\\.cmr|_ndvi\\.jpg)",
+  "url_path": "{cmrMetadata.Granule.GranuleUR}/{file.name}",
```

Comment:
The indentation looks off here if you could fix it real fast.

Reply:
I looked at that and I could no see a difference on my editor. But sure